geoportal/c2cgeoportal_geoportal/scaffolds/create/geoportal/vars.yaml_tmpl

```diff
@@ -161,6 +161,18 @@ vars:
     # - display: Child:
     #   url: {web_protocol}://{host}/child/wsgi
 
+  # The dogpile.cache configuration.
+  cache:
+    std:
+      arguments:
+        # Standalone version
+        # host: '{REDIS_HOST}'
+        # port: '{REDIS_PORT}'
+        # Openshift version
+        sentinels:
+          - - ${REDIS_HOST}
+              ${REDIS_PORT}
```

Comment:
Seems wreid, or I do not know this syntax `- -`

Reply:
This one is wrong, one dash is missing